sunpy/map/mapcube.py

```diff
@@ -24,6 +26,8 @@ class MapCube(object):
     """
     MapCube
 
+    Deprecated in favor of MapSequence. MapSequence has the same functionality as MapCube.
```

Comment:
I think the best thing to do is to put the deprecated decorator on the class. https://github.com/sunpy/sunpy/blob/master/sunpy/util/decorators.py#L20